dataRetriever/mock/peerListCreatorStub.go

```diff
@@ -4,14 +4,20 @@ import "github.com/ElrondNetwork/elrond-go/p2p"
 
 // PeerListCreatorStub -
 type PeerListCreatorStub struct {
-	PeerListCalled func() []p2p.PeerID
+	PeerListCalled           func() []p2p.PeerID
+	IntraShardPeerListCalled func() []p2p.PeerID
 }
 
 // PeerList -
 func (p PeerListCreatorStub) PeerList() []p2p.PeerID {
 	return p.PeerListCalled()
 }
 
+// IntraShardPeerList -
+func (p PeerListCreatorStub) IntraShardPeerList() []p2p.PeerID {
```

Comment:
p *PeerListCreatorStub for all pointer receivers in this file?

Reply:
done